elements/pf-text-area/pf-text-area.css

```diff
@@ -0,0 +1,192 @@
+  --pf-c-form-control--m-icon-sprite__select--invalid--BackgroundPosition: calc(100% - var(--pf-global--spacer--md, 1rem) - var(--pf-global--spacer--lg));
+}
+
+textarea {
+  --pf-c-form-control--success--BackgroundPositionY: var(--pf-c-form-control--textarea--success--BackgroundPositionY);
+  --pf-c-form-control--invalid--BackgroundPositionY: var(--pf-c-form-control--textarea--invalid--BackgroundPositionY);
+  --pf-c-form-control--m-warning--BackgroundPositionY: var(--pf-c-form-control--textarea--m-warning--BackgroundPositionY);
+
+  width: var(--pf-c-form-control--textarea--Width);
+  height: var(--pf-c-form-control--textarea--Height);
+  vertical-align: bottom;
+  color: var(--pf-c-form-control--Color);
+  width: var(--pf-c-form-control--Width);
+  padding: var(--pf-c-form-control--PaddingTop) var(--pf-c-form-control--PaddingRight) var(--pf-c-form-control--PaddingBottom) var(--pf-c-form-control--PaddingLeft);
+  font-size: var(--pf-c-form-control--FontSize);
+  line-height: var(--pf-c-form-control--LineHeight);
+  background-color: var(--pf-c-form-control--BackgroundColor);
+  background-repeat: no-repeat;
+  border: var(--pf-c-form-control--BorderWidth) solid;
+  border-color: var(--pf-c-form-control--BorderTopColor) var(--pf-c-form-control--BorderRightColor) var(--pf-c-form-control--BorderBottomColor) var(--pf-c-form-control--BorderLeftColor);
+  border-radius: var(--pf-c-form-control--BorderRadius);
+  appearance: none;
+}
+
+textarea::placeholder {
+  color: var(--pf-c-form-control--placeholder--Color);
+}
+
+textarea:hover {
+  --pf-c-form-control--BorderBottomColor: var(--pf-c-form-control--hover--BorderBottomColor);
+}
+
+textarea:focus {
+  --pf-c-form-control--BorderBottomColor: var(--pf-c-form-control--focus--BorderBottomColor);
+
+  padding-bottom: var(--pf-c-form-control--focus--PaddingBottom);
+  border-bottom-width: var(--pf-c-form-control--focus--BorderBottomWidth);
+}
+
+textarea:invalid {
+  --pf-c-form-control--PaddingRight: var(--pf-c-form-control--invalid--PaddingRight);
+  --pf-c-form-control--BorderBottomColor: var(--pf-c-form-control--invalid--BorderBottomColor);
+
+  padding-bottom: var(--pf-c-form-control--invalid--PaddingBottom);
+  background-image: var(--pf-c-form-control--invalid--BackgroundUrl);
+  background-position: var(--pf-c-form-control--invalid--BackgroundPosition);
+  background-size: var(--pf-c-form-control--invalid--BackgroundSize);
+  border-bottom-width: var(--pf-c-form-control--invalid--BorderBottomWidth);
+}
+
+textarea:disabled {
+ --pf-c-form-control--BackgroundColor: var(--pf-c-form-control--disabled--BackgroundColor);
+  color: var(--pf-c-form-control--disabled--Color);
+  cursor: not-allowed;
+  border-color: var(--pf-c-form-control--disabled--BorderColor);
+}
+
+textarea[readonly] {
+  background-color: var(--pf-c-form-control--readonly--BackgroundColor);
+}
+
+.vertical { resize: vertical; }
+.horizontal { resize: horizontal; }
+
```

Comment:
```Suggestion
.both { resize: both; }
```
The resize property takes a value of `both` and there is a patternfly modifier `.pf-m-resize-both`